index.js

```diff
@@ -1 +1,50 @@
-console.log("Code your solution!")
+let todoForm = document.querySelector('#todo-item');
+
+todoForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    let userList = document.querySelector('ul');
+    let userInput = event.target['list-item'].value;
+    if (!userInput) {
+        let p = document.createElement('p');
+        p.textContent = 'Error. Todo cannot be empty';
+        todoForm.after(p);
+    } else {
+        let listItem = document.createElement('li');
+        listItem.textContent = userInput;
+        userList.append(listItem);
+        
+        // let deleteButton = document.createElement('button');
+        // deleteButton.setAttribute('type', 'submit');
+        // deleteButton.textContent = 'Delete';
+        // listItem.append(deleteButton);
+        event.target['list-item'].value = "";
+        // deleteButton.addEventListener('click', () => {
+        //     listItem.remove();
+        // });
```

Comment:
I think I commented this part out because it caused one of the tests to fail, but it did work. I didn't understand why it caused it to fail. I will figure it out, but thanks! I'll schedule an office hour if anything.